src/chains/mainnet/vm/precompiles.ts

```diff
@@ -68,7 +67,9 @@ export const precompiles: Precompile[] = [
         description: 'The SHA-256 hash of the input data',
       },
     ],
-    references: [],
+    references: [
+      'https://github.com/ethereum/execution-specs/blob/master/src/ethereum/shanghai/vm/precompiled_contracts/sha256.py',
```

Comment:
For each mainnet precompile could you include all 3 `ethereum/execution-specs` references, e.g. the ecReover precompile links to the files in the execution-spects defining it's (1) implementation, (2) min gas cost, and (3) address, which are in different files